bundler/lib/bundler/cli/gem.rb

```diff
@@ -184,10 +184,12 @@ def run
         config[:linter_version] = rubocop_version
         Bundler.ui.info "RuboCop enabled in config"
         templates.merge!("rubocop.yml.tt" => ".rubocop.yml")
+        config[:linter_config_path] = ".rubocop.yml "
```

Comment:
I understand this is following previous approach in `ci_config_path`, but isn't it a bit weird to include a trailing space, and then interpolate in this the gemspec list without leaving any separation from other elements in that list?

Should we fix `ci_config_path` to do something simpler, instead? Not really necessary for this PR, since this is following the previous approach after all, but feel free to improve this if you want!